### PR TITLE
Rebased 94

### PR DESCRIPTION
**Pull request by nniehoff**
Rebasing #94 from master to hopefully pass linting.

### CODE REVIEW

**Comment by nniehoff** [comment]
@nunomcruz @luqasz I rebased this PR in my fork as I could really use this feature, I have also tested this on a `CRS328-24P-4S+`, works well.  Thanks, hope to see this feature soon.

File: napalm_ros/ros.py [hunk 1]
```diff
@@ -363,6 +364,19 @@ def get_facts(self):
             ),
         }
 
+    def get_config(self, retrieve='all', full=False, sanitized=False):
```
Comment (nniehoff) [inline napalm_ros/ros.py:367]: Should the defaults here be True?

```suggestion
    def get_config(self, retrieve='all', full=True, sanitized=True):
```

Comment (luqasz) [reply napalm_ros/ros.py:367]: No.

**Comment by nniehoff** [comment]
@luqasz thanks, done

**Pull request merged**